compiler-rt/test/msan/lit.cfg.py

```diff
@@ -18,6 +18,11 @@
     ]
     + [config.target_cflags]
     + config.debug_info_flags
+    + (
+        ["--sysroot", config.cmake_sysroot]
+        if config.cmake_sysroot and config.host_os == "Linux"
+        else []
+    )
```

Comment:
I'd move this to a separate `if` block similar to other blocks below for readability.